test/templates/modal/index.njk

```diff
@@ -1,17 +1,14 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
 {% from "./component.njk" import modal %}
-{% from "../icon/component.njk" import icon %}
 
 {% block content %}
 
-
 <p>
-NOTE: The <code>modal</code> itself does not provide the outer container. Use the <code>.fd-overlay</code> layout component in conjunction with the <code>.fd-ui__overlay</code> template container to hide and show the modal. See <a href="http://localhost:3030/pages/overlay">the overlay example</a>.
-
+  NOTE: The <code>modal</code> itself does not provide the outer container. Use the <code>.fd-overlay</code> layout component in conjunction with the <code>.fd-ui__overlay</code> template container to hide and show the modal. See <a href="http://localhost:3030/pages/overlay">the overlay example</a>.
```

Comment:
What the heck does this sentence even mean?  Seems like we could get rid of that in the playground.